packages/server/database/migrations/20230116165620-fixStarfishTemplate.ts

```diff
@@ -0,0 +1,41 @@
+import {R} from 'rethinkdb-ts'
+
+export const up = async function (r: R) {
+  const now = new Date()
+  const missingLessOfPrompt = {
+    createdAt: now,
+    description: 'What are we over-doing or doing too much of?',
+    groupColor: '#D9D916',
+    id: 'starfishTemplate:lessOfPrompt',
+    isActive: true,
+    question: 'Less Of ➖',
+    sortOrder: 1,
+    teamId: 'aGhostTeam',
+    templateId: 'starfishTemplate',
+    title: 'Less Of ➖',
+    updatedAt: now
+  }
+  const missingMoreOfPrompt = {
+    createdAt: now,
+    description: 'What are we not taking enough advantage of?',
+    groupColor: '#45E5E5',
+    id: 'starfishTemplate:moreOfPrompt',
+    isActive: true,
+    question: 'More Of ➕',
+    sortOrder: 2,
+    teamId: 'aGhostTeam',
+    templateId: 'starfishTemplate',
+    title: 'More Of ➕',
+    updatedAt: now
+  }
+
+  await r.table('ReflectPrompt').insert([missingLessOfPrompt, missingMoreOfPrompt]).run()
```

Comment:
+1 what we should do here is
```ts
  await r.table('ReflectPrompt').insert([missingLessOfPrompt, missingMoreOfPrompt], {conflict: 'error'}).run()
```
This would have avoided the error in the past